### ISSUE TITLE
No way to delete annotations except to drag them outside the image window

### ISSUE THREAD

**Issue by ezhang816** [OrionMD/pacsbin-community#22]
Hi Jason,

There is no way to remove arrows, ROIs and other annotations except to drag them outside the image window. Selecting the annotation and pressing delete does not work.

Thanks!

**Comment by jmhmd** [comment]
Yes, that's the only way to delete annotations currently. Would also like this to be more intuitive.

**Comment by jmhmd** [comment]
An "eraser" tool has been added in v2.

Still investigating how it would work to use the delete key, probably possible.

**Comment by jmhmd** [comment]
Delete key support in commit 74d6f2b5612301158ad6c3cca5600a8a3101a795
